utils/include/ThreadSafeVariable.h

```diff
@@ -0,0 +1,32 @@
+#pragma once
+
+#include <mutex>
+
+template <typename T>
+class ThreadSafeVariable 
```

Comment:
Fix this file according to code style, or it was already fixed just pull those changes